Customization/T250/CodeSnippets/Step1.5.2/RS401000.aspx

```diff
@@ -36,7 +36,7 @@
           <Columns>
               <px:PXGridColumn DataField="OrderType" Width="70" />
               <%-- The modified code --%>
-              <px:PXGridColumn DataField="OrderNbr" Width="72"
```

Comment:
we got rid of explicit width declaration of the fields several releases ago

Reply:
But we have Width specified for all other columns in this grid. Should we remove Width for them as well?

Also, if this is a general rule, we need to create a task for update of all of the courses for this issue, because I'm pretty sure we have Width explicitly specified in some courses.

Reply:
Yes. Stanislav Lesin said, "It would be better to remove explicit Width from all the courses, since it is some useless information, not really needed in a regular course"